plugins/filetrees/nvim-tree.nix

```diff
@@ -365,10 +365,6 @@ in {
           Increase if you experience performance issues around screen refresh.
         '';
 
-        hideRootFolder = helpers.defaultNullOpts.mkBool false ''
-          Hide the path of the current working directory on top of the tree.
-        '';
-
```

Comment:
Please, mark the option as deprecated using `mkRemovedModule`.
Example here: https://github.com/nix-community/nixvim/blob/73f21c063dac16340c60d99a143dd3c7c07a1e70/plugins/filetrees/neo-tree.nix#L13-L24